src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+function gallery() {
+  const list = document.querySelector('#thumbs');
+  const image = document.querySelector('#largeImg');
+
+  list.addEventListener('click', (thisEvent) => {
+    thisEvent.preventDefault();
+
+    const link = thisEvent.target.closest('a').getAttribute('href');
```

Comment:
There is a potential issue here if `thisEvent.target.closest('a')` returns null, which would happen if the click event is not on an anchor element or its descendant. This would cause `getAttribute('href')` to throw an error. Consider adding a check to ensure that `closest('a')` returns a valid element before attempting to access its attributes.